templates/regression/test_add.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+
+{% block title %}Regression Test Add {{ super() }}{% endblock %}
+{% block body %}
+    {{ super() }}
+    <br />
+    <div class="grid-x">
+        <div class="callout warning">
+            <h5>Regression Test Add</h5>
+            <p>Please add new regression test</p>
+        </div>
+    </div>
+    <form method="post" name="AddTestForm" id="AddTestForm" action="{{ url_for('.test_add') }}">
+        {{ form.csrf_token }}
+        {% with messages = get_flashed_messages(with_categories=true) %}
+            {% if messages %}
+                <div class="grid-x">
+                    <div class="callout alert">
+                        <ul class="flashes">
+                            {% for category, message in messages %}
+                            <li class="{{ category }}">{{ message }}</li>
+                            {% endfor %}
+                        </ul>
+                    </div>
+                </div>
+            {% endif %}
```

Comment:
you are not printing form errors
                    {% if form.errors %}
                        <div class="grid-x">
                            <div class="callout alert">
                                {% for field, error in form.errors.items() %}
                                    {% for e in error %}
                                        {{ e }}<br>
                                    {% endfor %}
                                {% endfor %}
                            </div>
                        </div>
                    {% endif %}